code/modules/holodeck/turfs.dm

```diff
@@ -70,24 +70,24 @@
 	gender = PLURAL
 	name = "sand"
 	desc = "This is better than a vacation, since you're still getting paid."
-	icon = 'icons/misc/beach.dmi'
+	icon = 'icons/turf/sand.dmi'
 	icon_state = "sand"
 	bullet_bounce_sound = null
 	tiled_dirt = FALSE
 
-/turf/open/floor/holofloor/beach/coast_t
+/turf/open/floor/holofloor/beach/coast
```

Comment:
Your path changes are going to need an UpdatePaths script.

Reply:
You're so right